infer-web.py

```diff
@@ -70,6 +58,10 @@ def forward_dml(ctx, x, scale):
 mem = []
 if_gpu_ok = False
 
+
+
+start_websocket_server('127.0.0.1', 15159)
```

Comment:
You can also add this to config and add a commandline parameter to control this.